tools/generate_project_statistics_csv.php

```diff
@@ -1,27 +1,17 @@
 #!/usr/bin/env php
-<?php
+<?php declare(strict_types=1);
 /**
- * @author Stella Lee
- * Date: 15-08-15
+ * Generates a CSV report of various statistics about this study.
  *
- * Project Statistics - Updating Counts From Each LORIS
```

Comment:
I'd suggest keeping a line that documents what the script is for.

Reply:
If you look to the right I added a new one.